packages/auto-ensure/package.json

```diff
@@ -24,7 +24,8 @@
     "check-update": "cliff-jumper --dry-run"
   },
   "dependencies": {
-    "@joshdb/middleware": "1.1.0-next.afc5762.0"
+    "@joshdb/middleware": "1.1.0-next.afc5762.0",
+    "@sapphire/utilities": "^3.6.2"
```

Comment:
I think there's a 3.7 at least, can you update this

Reply:
I'll let renovate do it, not like we need the version?